apps/settings/elements/root.html

```diff
@@ -24,37 +24,37 @@ <h2 data-l10n-id="networkAndConnectivity">Network &amp; Connectivity</h2>
           </label>
         </li>
         <li data-category="connectivity">
-          <a id="menuItem-wifi" class="menu-item" data-icon="wifi-4" href="#wifi">
+          <a id="menuItem-wifi" class="menu-item" data-icon="wifi-4" href="#wifi" data-l10n-id="wifiSection" aria-describedby="wifi-desc">
```

Comment:
Also since the aria-label will override all of the link's content text, it will not read the aria-label + main text in content (twice) and will only read the aria-label.